addons/medical_gui/XEH_postInit.sqf

```diff
@@ -76,3 +78,20 @@ GVAR(selfInteractionActions) = [];
 ["ace_interactMenuClosed", {
     QGVAR(RscPatientInfo) cutFadeOut 0.3;
 }] call CBA_fnc_addEventHandler;
+
+[QEGVAR(medical,woundReceived), {
+    params ["_unit", "_allDamages", ""];
+    if !(GVAR(peekMedicalOnHit) && ACE_PLAYER == _unit) exitWith {};
+
+    private _bodypart = toLower (_allDamages select 0 select 1);
```

Comment:
There're only ANSI chars, right ?
```suggestion
    private _bodypart = toLowerANSI (_allDamages select 0 select 1);
```

Reply:
Probably, but I was just copying the existing usage from here:
https://github.com/acemod/ACE3/blob/78bb0514d1233712428da023cdc732c037ed8aeb/addons/medical_damage/functions/fnc_woundsHandlerBase.sqf#L44

Reply:
Ya, seen exactly that one before and it annoys me a lot.

Reply:
Why this? We don't do it anywhere else.